README.md

```diff
@@ -119,13 +119,19 @@ The command above mounts both of these directories.
 ## Running as an executable
 Supported only on x86_64!
 
-Run the following to have gprofiler running continuously, uploading to Granulate Performance Studio:
+Run the following to have gprofiler running continuously, in the background, uploading to Granulate Performance Studio:
 ```bash
 wget https://github.com/Granulate/gprofiler/releases/latest/download/gprofiler
 sudo chmod +x gprofiler
-sudo ./gprofiler -cu --token <token> --service-name <service> [options]
+sudo sh -c "setsid ./gprofiler -cu --token <token> --service-name <service> [options] > /dev/null 2>&1 &"
 ```
 
+You can verify that it has started correctly with `pgrep gprofiler`. If that doesn't find any process, try running without `> /dev/null 2>&1 &` so you can inspect the output, and look for errors.
+
+For non-daemon mode runes, you can remove the `setsid` and `> /dev/null 2>&1 &` parts.
+
+The logs can then be viewed in their default location (`/var/log/gprofiler`).
+
```

Comment:
I think you should add the `pgrep` command to the commands list above, I can assure you some people will not read this part and will only run the commands. Adding the `pgrep` above will help mitigate that

Reply:
K will reorder (tho my intention was not as a pre-check but merely as a way to know if it's running - since it starts as a daemon, you have no feedback whether it has even started or not)